get rid of treatment location profile, and add location references to examples

saulakravitz opened the pull request.

saulakravitz committed:
get rid of treatment location profile, and add location references to examples

mvonsieb commented:
Please remove also from CapabilityStatement. The example references do not resolve yet. See https://build.fhir.org/ig/HL7/codex-radiation-therapy/branches/nukelocation/qa.html

saulakravitz committed:
fix treatment location

saulakravitz committed:
fix treatment location

saulakravitz committed:
delete Treatment Location profile

saulakravitz committed:
update CapStmt

saulakravitz committed:
update warnings

saulakravitz commented:
Fixed object references.   Sushi (intentionally) doesn't flag these, which always causes me grief.

Updated CapStmt to require USCoreLocation with same query interface.

mvonsieb commented:
Good to merge.

saulakravitz committed:
Merge branch 'master' into nukelocation
